### ISSUE TITLE
Tablet Issue: App Not Searchable on App Store/Google Play & V1.7.0 not available on Android 4.0 tablets 

### ISSUE THREAD

**Issue by jparlina** [pg-irc/pathways-frontend#930]
A user reported that Android 4.0 ([Rumie tablets](https://www.rumie.org/wp-content/uploads/2018/07/2018.01-RumieTabletSpecSheet-Protonv1.01.pdf)) are only pulling older versions (V1.0.13) of Arrival Advisor from Google Play, not the latest  V1.7.0. Tested on Android 4.2 tablet - Arrival Advisor is not listed on Google Play. Checked on Google Analytics, the lowest version of Android running Arrival Advisor v1.7.0. is Android 5.0. 

Searched "Arrival Advisor" on Google Play on Android 4.2 tablet and on App Store on iPad 10.3.3 - NO RESULT (see screenshots below). However, when user go to Arrival Advisor website, and click on the link to download the app, user was able to download the latest version V1.7.0 on the iPad (will need to test on Android tablet). 

![IMG_3711.jpg](https://images.zenhubusercontent.com/5ceeccc646b329310b95bccd/3b89a612-0508-41ad-991f-3f6ea91d4bba)

### Acceptance Criteria 
- [ ]  download V1.7.0. on Android devices lower than Android 5.0 
- [ ]  can find and download Arrival Advisor on Google Play and App Store when searching on tablets

### Steps for QA: 
1.  Use Android device with OS lower than 5.0
2. Go to Google Play store
3. Find Arrival Advisor
4. Check version
5. Download the app

**Comment by ian-lin-PG** [comment]
The current up to date versions of `Expo supports Android 5+ and iOS 10+.` according to the Expo faq found here https://docs.expo.io/versions/latest/introduction/faq/. In order to support older version of android we must roll back to at least Expo sdk 31.0.0 in order to support [`iOS 9 or Android 4.4`](https://blog.expo.io/expo-sdk-v31-0-0-is-now-available-cad6d0463f49). To support even older versions of iOS and Android we would have to roll back even more.

**Comment by ArtiBPG** [comment]
I dont think we want to downgrade but we can probably list compatibility:

https://support.google.com/googleplay/android-developer/answer/7353455?hl=en

**Comment by rasmus-storjohann-PG** [comment]
@ian-lin-PG do some research on what we can do to communicate this issue to our users.

**Comment by tomy-pg** [comment]
I did some testing with an older version of Android on an Android emulator. 
The version of Android I used was 7.0 as it was the lowest the Android Studios would offer with a simulator that made the Google Play Store available. Similar testing could not be done with an iOS simulator because Xcode does not make the App Store available on their emulators. The store being available is important to be able to see how apps handle compatibility issues on the store. 

I reviewed apps that do not support Android versions 7.0 or lower because of the restrictions I just outlined. I looked at Netflix, Disney+ and Uber. All three apps handled compatibility issues the same way. 

**On the Google Play Store:**

All three of these apps detail the minimum version of Android OS needed to use their app. Arrival Advisor also has this information available on the Google Play store. However, if a device cannot support their app, the app does not show up on the store. So this information won't be readily available on the store for users with these devices. 

To actually find the app on a device that does not support it, users have to go on a browser and type in Netflix app to find it. When users try downloading the app they are greeted with a message saying: 

Your device isn’t compatible with this version. 

<img width="287" alt="Screen Shot 2019-12-17 at 4 27 18 PM" src="https://user-images.githubusercontent.com/47794846/71114206-221dcf00-2184-11ea-885c-cee0a3f9a07f.png">

**FAQ / Help**
Outside of the store all three apps also communicate the software requirements to their users on their FAQ / Help pages on their websites. I think we could follow this example and add the software requirements to the Arrival Advisor website. We could also provide a list of compatible devices on the FAQ page as well. 

https://help.netflix.com/en/node/12983
https://help.disneyplus.com/csp?id=csp_article_content&sys_kb_id=736b776cdbcd04d03c0cf158bf961908
https://help.uber.com/partners/article/downloading-the-driver-app-android--?nodeId=e75df75e-552a-4072-a852-d2e981566bdd

As @ian-lin-PG mentioned, since we are using the most up to date Expo we support devices with Android 5+ and iOS 10+. Rolling back or making another version of the app with an older SDK would be a short term solution because these SDK's are deprecated within 6 months which would not allow us to generate new Expo builds.
